### PR TITLE
Add Info Popup w/ 'App Version' menu item integration.

### PR DESCRIPTION
**Pull request by swoocn**
_Clicking on [About Map of Pi] -> [App Version] menu item will toggle the appropriate Info popup._

image

**[see commits]**

### CODE REVIEW

File: src/components/shared/Info/Info.module.css [hunk 1]
```diff
@@ -1,116 +1,99 @@
...
+}
+
+.social_media__link,
+.social_media__icon {
+  padding: 2px; /* Increases the touchable area */
+  margin: 0 4px; /* Adjusts spacing around links */
+}
+
+.social_media__link {
+  /* display: inline-flex;
```
Comment (zolcsi) [inline src/components/shared/Info/Info.module.css]: it seems we don't need this code. let's delete it :)

File: src/components/shared/sidebar/sidebar.tsx [hunk 2]
```diff
@@ -62,6 +66,9 @@ function Sidebar(props: any) {
       code === 'dark' ? setTheme('dark') : setTheme('light');
       props.setToggleDis(false);
     }
+    if (title === 'About Map of Pi' && code === 'version') {
```
Comment (zolcsi) [inline src/components/shared/sidebar/sidebar.tsx:69]: will this get along with internationalization?

**Review by zolcsi — APPROVED** [review]
Looking good. Two minor comments fyi

File: src/components/shared/sidebar/sidebar.tsx (hunk 2 above)
Comment (swoocn) [reply src/components/shared/sidebar/sidebar.tsx:69]: Great point!  
There will be a separate task to internationalize the contents.

**Pull request merged**